File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int i = 1; i <= height ; i++)
    {
        for(int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for(int k = i; k > 0; k--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    size = get_int("Size of Pyramid (Size must be between 1 and 8): ")
    if size >= 1 and size <= 8:
        break

for i in range(size):
    print(" " * (size - 1 - i), end="")
    print("#" * (i + 1) + "  " + "#" * (i + 1))


